Sources/Authgear.swift

```diff
@@ -830,6 +981,31 @@ public class Authgear: NSObject {
         }
     }
 
+    public func refreshIDToken(handler: @escaping VoidCompletionHandler) {
```

Comment:
I think we need to call `refreshAccessTokenIfNeeded` and ensure the access token is refreshed. We can test it by calling `refreshIDToken` right after the app is just launched.